editoast/src/core/conflict_detection.rs

```diff
@@ -75,10 +75,20 @@ pub struct ConflictDetectionResponse {
     pub conflicts: Vec<Conflict>,
 }
 
+#[derive(Debug, Clone, PartialEq, Deserialize, Serialize, ToSchema)]
+pub struct PacedTrainsConflicts {
+    paced_train_id: i64,
+    index: i64,
+}
+
 #[derive(Debug, Clone, PartialEq, Deserialize, Serialize, ToSchema)]
 pub struct Conflict {
-    /// List of train ids involved in the conflict
-    pub train_ids: Vec<i64>,
+    /// List of train schedule ids involved in the conflict
+    pub train_schedule_ids: Vec<i64>,
+    /// List of paced train ids involved in the conflict
+    #[serde(default)]
+    #[schema(inline)]
+    pub paced_trains: Vec<PacedTrainsConflicts>,
```

Comment:
We have `train_schedule_ids` (above), that reference a train schedule (it is not the train schedule data, only a reference to one). `paced_trains` is also only a reference to an occurrence of a paced train (it's not the data of this paced train). So I would keep the naming similar. What about `paced_train_occurrence_ids`?